lib/livebook_web/live/session_live/add_file_entry_file_component.ex

```diff
@@ -48,7 +47,12 @@ defmodule LivebookWeb.SessionLive.AddFileEntryFileComponent do
   end
 
   def update(assigns, socket) do
-    {:ok, assign(socket, assigns)}
+    {file, assigns} = Map.pop!(assigns, :file)
```

Comment:
Why are we using the notebook file as the starting point here? It's about adding a file entry from any file in the file system, so default dir is a better default.